extensions/blocks/contact-form/edit.js

```diff
@@ -323,6 +324,7 @@ function JetpackContactFormEdit( {
 				{ ! isSimpleSite() && (
 					<CRMConnectionSettings jetpackCRM={ jetpackCRM } setAttributes={ setAttributes } />
 				) }
+				{ ! isSimpleSite() && <NewsletterIntegrationSettings /> }
```

Comment:
I think we could simplify things a bit here.
 
```suggestion
				{ ! isSimpleSite() && (
					<CRMConnectionSettings jetpackCRM={ jetpackCRM } setAttributes={ setAttributes } />
					<NewsletterIntegrationSettings />
				) }
```

Reply:
This logic is updated